journals/journal_abbreviations_acs.csv

```diff
@@ -863,7 +865,7 @@ Journal of Chemical Ecology;J. Chem. Ecol.
 Journal of Chemical Education;J. Chem. Educ.
 Journal of Chemical Engineering of Japan;J. Chem. Eng. Jpn.
 Journal of Chemical Information and Modeling;J. Chem. Inf. Model.
-Journal of Chemical Physics;J. Chem. Phys.
+The Journal of Chemical Physics;J. Chem. Phys.
```

Comment:
Please sort alphabetically ^^.

Reply:
The script I was using to merge several lists sorted by abbreviation, I now ran a sort by journal name - though that moved quite a lot of entries around. Better?